src/Tizen.TV.UIControls.Forms/Renderer/MediaPlayerImpl.cs

```diff
@@ -30,22 +30,27 @@ namespace Tizen.TV.UIControls.Forms.Renderer
 {
     public class MediaPlayerImpl : IPlatformMediaPlayer
     {
-        Player _player;
+        protected Player _player;
```

Comment:
Use property, 
field should be private
```suggestion
        Player _player;
        protected Player Player => _player;
```